inputs/diffusion/conduction.in

```diff
@@ -74,8 +74,7 @@ siefloor = 1.0e-15
 type = constant 
 cond = 1.0e-1
 
-<gravity>
-type = uniform
+<gravity/uniform>
```

Comment:
Should this be in this input file?

Reply:
what do you mean? This has uniform gravity

Reply:
I read defaults zero... but it looks like the pgen does invoke gravity (as does the CI tests)... disregard